docs/samples/browser-plugin-meetings/app.js

```diff
@@ -2823,6 +2827,21 @@ const participantsList = document.querySelector('#participant-list');
 const participantTable = document.querySelector('#participant-table');
 const participantButtons = document.querySelector('#participant-btn');
 
+participantTable.addEventListener('click', (event) => {
+  if (event.target.type === 'radio') {
+    const selectedParticipant = meeting.members.membersCollection.get(event.target.id);
+    if (!selectedParticipant) {
+      return;
+    }
+    const muteButton = document.getElementById('mute-participant-btn')
+    if (selectedParticipant.isAudioMuted) {
+      muteButton.innerText = meeting.selfId === selectedParticipant.id ? 'Unmute' : 'Request to unmute';
```

Comment:
there is also an option to unmute someone if "moderated unmute mode" is used, but it's not a blocker for this PR, can be added later